project.clj

```diff
@@ -84,13 +84,13 @@
                              [lein-cljsbuild "1.1.7"]
                              [lein-environ "1.1.0"]
                              [lein-figwheel "0.5.16"]
-                             [lein-virgil "0.1.8"]]
+                             #_[lein-virgil "0.1.8"]]
```

Comment:
Was this really the cause of slowdown?

Reply:
It's definitely slow regardless. I'll put it back.